cmd/chisel/cmd_info.go

```diff
@@ -0,0 +1,138 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/jessevdk/go-flags"
+	"gopkg.in/yaml.v3"
+
+	"github.com/canonical/chisel/internal/setup"
+)
+
+var shortInfoHelp = "Show information about package slices"
+var longInfoHelp = `
+The info command shows detailed information about package slices.
+
+It accepts a whitespace-separated list of strings. The list can be
+composed of package names, slice names, or a combination of both. The
+default output format is YAML. When multiple arguments are provided,
+the output is a list of YAML documents, separated by a line
+consisting of three dashes ("---").
```

Comment:
"documents, separated by a line consisting of three  dashes ("---")."

=>

"documents separated by a "---" line."